versions/factory/errors.go

```diff
@@ -0,0 +1,9 @@
+package factory
+
+import "errors"
+
+// ErrFileNotFound signals that the provided path is invalid and does not belong to a file or a directory
+var ErrFileNotFound = errors.New("file not found")
```

Comment:
Is this used?

Reply:
no.. I thought I'll treat this error, but it already treated on `core.OpenFile()`. removed